opt/system-setup.py

```diff
@@ -30,7 +30,7 @@ def debian_compat(self):
         self.install("gawk")
```

Comment:
I know this sounds silly - but let's just add all of the self.install to a single call:
```
self.install('libssl-dev python3-regex python3-networkx python3-numpy')
```

The reason, is that it calls apt once, meaning this will run faster.